docs/get-support.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Get Support
+---
+
+# Get Event Bus Support
+
+The best way to request support from the Event Bus Team is to reach out to us via the [#ves-event-bus](https://dsva.slack.com/archives/C042ZQ7JUAX) Slack channel in the Digital Service VA workspace.
+
+Please avoid direct messaging an Event Bus team member in Slack. Chatting openly in the [#ves-event-bus](https://dsva.slack.com/archives/C042ZQ7JUAX) channel allows others to learn from the questions and answers discussed.
+
+# Slack Channel Support Hours 
+Event Bus Slack channel support hours are from 9am - 5pm ET, Monday through Friday, with the exception of Federal holidays.
+
+- Response times are based on the nature of the posting:
+- Incidents (outage or error): ASAP. Please clearly indicate this is an “INCIDENT” in the post.
```

Comment:
Perhaps the two items under response types could be listed separately as sub-items, i.e.:

- Response times ...
    - Incidents
    - General inquiries

Reply:
Oops! Its suppose to be

Response Time
- Incident
- General Inquiries

Thank you for the catch! I also notice the format is a bit funky as I continue to stare.. Will update testing